examples/arrays/arrays.bal

```diff
@@ -6,10 +6,25 @@ public function main() {
     io:println(a.length());
 
     // Creates an array specifying an array literal.
-    int[] b = [1, 2, 3, 4, 5, 6, 7, 8];
+    int[] b = [1, 3, 2, 4, 7, 6, 5, 8];
     io:println(b[0]);
     io:println(b.length());
 
+    // Array supports several inbuilt functions such as `.reverse()`, `.pop()`, `.push()` and `.removeAll()`
+    int[] bReveresed = b.reverse();
+    io:println("Reversed: ", bReveresed.toString());
```

Comment:
We don't need to say `.toString()` since `io:println()` has a rest param of type `any|error`.

Let's fix everywhere.

Reply:
Fixed